src/de/jost_net/JVerein/Einstellungen.java

```diff
@@ -226,6 +228,45 @@ public static void setEinstellung(Einstellung einst)
     einstellung = einst;
   }
 
+  public static GregorianCalendar getBeginnGeschaeftsjahr(
```

Comment:
Das sieht mir überkompliziert aus. Was genau soll die Funktion eigentlich machen?
In Java arbeitet man eigentlich nicht direkt mit dem "GregorianCalender" und instanziert den auch nicht direkt. Man verwendet (wenn man die Calendar-Implementierung nutzen möchte) "Calendar.getInstance()".

Reply:
Grundsätzlich gebe ich dir Recht - ich würde aber auch ungerne einen funktionieren Code, welcher von [FullHD23](https://github.com/FullHD23 in https://github.com/openjverein/jverein/pull/24 schon mal verified wurde, komplett umbauen.